app/models/manageiq/providers/openstack/manager_mixin.rb

```diff
@@ -70,7 +70,7 @@ def connect_identity
 
   def event_monitor_options
     @event_monitor_options ||= begin
-      opts = {:ems => self}
+      opts = {:ems => self, :automatic_recovery => false, :recover_from_connection_close => false}
```

Comment:
You pass `false` here for these two settings, but then set them to `true` explicitly above.  Is that on purpose?